ibm_catalog.json

```diff
@@ -336,6 +404,21 @@
                 }
               ]
             },
+            {
+              "key": "logs_routing_tenant_regions",
```

Comment:
why are you adding this? This is not used as the Cloud Logs DA is not a dependant DA (its the other way around)